### PR TITLE
Update maintainers and codeowners

### PR DESCRIPTION
**Pull request by bitwiseguy**
This PR serves as a voting ballot for changes in the maintainer team. As stated in the baseline docs, a 2/3 majority vote is needed for these changes to pass. Only votes from current maintainers will be considered valid. There are currently 7 maintainers, so at least 5 must approve for the changes to be accepted.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ Maintainers are people who take an active role in writing code that advances the
 
 Code and Systems maintainers
   - [Sam Stokes](https://github.com/bitwiseguy) 
-  - [Patrick Macom](https://github.com/pmacom)
   - [Kartheek Solipuram](https://github.com/skarred14)
   - [Brian Chamberlain](https://github.com/breakpointer)
   - [Kyle Thomas](https://github.com/kthomas)
   - [George Spasov](https://github.com/Perseverance)
   - [Tomasz Stanczak](https://github.com/tkstanczak)
+  - [Lucas Rodriguez](https://github.com/LucasRodriguez)
```
Comment (skarred14) [inline MAINTAINERS.md:25]: Lucas Rodriguez Benitez

**Review by skarred14 — DISMISSED** [review]

**Review by kthomas — DISMISSED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (bitwiseguy) [reply MAINTAINERS.md]: Fixed

**Review by skarred14 — APPROVED** [review]

**Review by breakpointer — APPROVED** [review]
Approving vote

**Review by kthomas — APPROVED** [review]

**Review by tkstanczak — APPROVED** [review]

**Comment by kthomas** [comment]
Measure passed.

**Pull request merged**